### PR TITLE
Fixing some grammar mistakes in markdown files.

### PR DESCRIPTION
**Pull request by VutonDesign**
After examining the "CONTRIBUTING" & "README" files, I found several grammar mistakes in them. 

## Things Fixed
- Fixed "road map" (It should all be one word)
- Adding necessary comma
- Capitalizing "url"
- Fixing "webclient" (Separate words) 
- Capitalizing another word

## Benefits of these fixes
- Makes the files simpler to read 
- Some sentences make more sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ These currently exist in static/js/lib
...
+During development, `npm start` watches any update you make to files in either `./static` or `./src` and triggers a rebuild of the project.  In development, the build is stored in memory, and not served from the `./build` directory.
 
 When running `npm start`, here are some important log messages to keep an eye out for:
-* `webpack: bundle is now VALID.` – the bundle has been loaded into memory and is now viewable in the browser. This will show up both once `npm start` has completed its setup, and also once updates you make to files have been re-compiled for viewing in the browser.
+* `webpack: bundle is now VALID.` – The bundle has been loaded into memory and is now viewable in the browser. This will show up both once `npm start` has completed its setup, and also once updates you make to files have been re-compiled for viewing in the browser.
 * `webpack: bundle is now INVALID.` – if you see this, then it means you have made updates to files that are still being compiled for browser viewing. Pages will still be viewable, but they will not see any updates you made yet.
```
Comment (towerofnix) [inline README.md]: If the above line is capitalized, shouldn't this one be, too?

Comment (VutonDesign) [reply README.md]: Ah, Yeah

Comment (duckboycool) [reply README.md]: Well if in the line below isn't.

Comment (VutonDesign) [reply README.md]: @towerofnix Fixed that 👍

**Comment by VutonDesign** [comment]
@thisandagain Yay, got the thisandagain stamp of approval! lol

**Pull request merged**